unit_tests/UnitTestHelperObjects.h

```diff
@@ -80,6 +81,56 @@ struct HelperObjects {
   sierra::nalu::AssembleElemSolverAlgorithm* assembleElemSolverAlg;
 };
 
+struct TpetraHelperObjects {
+  TpetraHelperObjects(stk::mesh::BulkData& bulk, stk::topology topo, int numDof, stk::mesh::Part* part)
+  : yamlNode(unit_test_utils::get_default_inputs()),
+    realmDefaultNode(unit_test_utils::get_realm_default_node()),
+    naluObj(new unit_test_utils::NaluTest(yamlNode)),
+    realm(naluObj->create_realm(realmDefaultNode, "multi_physics", false)),
+    eqSystems(realm),
+    eqSystem(eqSystems),
+    linsys(new sierra::nalu::TpetraLinearSystem(realm, numDof, &eqSystem, nullptr)),
+    assembleElemSolverAlg(nullptr)
+  {
+    realm.metaData_ = &bulk.mesh_meta_data();
+    realm.bulkData_ = &bulk;
+    eqSystem.linsys_ = linsys;
+
+    assembleElemSolverAlg = new sierra::nalu::AssembleElemSolverAlgorithm(realm, part, &eqSystem, topo.rank(), topo.num_nodes());
```

Comment:
Can we have a `TpetraHelperObjectsBase` and then specializations for elem, face elem, and edge algs? Currently we create an `assembleElemSolverAlg` instance even if we are testing FaceElem or Edge algorithms.